From 6ffb169dbe331627d70da3f79f928057f1a4e163 Mon Sep 17 00:00:00 2001
From: Hood Chatham <roberthoodchatham@gmail.com>
Date: Sun, 26 Dec 2021 07:34:40 -0800
Subject: [PATCH] Rename _page_trend_test.py to prevent test unvendoring

unvendor_tests will unvendor any file that ends in _test.py. 
Prevent that by changing the name of this file.

---
 scipy/stats/__init__.py                                   | 2 +-
 scipy/stats/{_page_trend_test.py => _page_trend_test_.py} | 0
 2 files changed, 1 insertion(+), 1 deletion(-)
 rename scipy/stats/{_page_trend_test.py => _page_trend_test_.py} (100%)

diff --git a/scipy/stats/__init__.py b/scipy/stats/__init__.py
index f410f5410..ad702c38d 100644
--- a/scipy/stats/__init__.py
+++ b/scipy/stats/__init__.py
@@ -453,7 +453,7 @@ from ._bootstrap import bootstrap
 from ._entropy import *
 from ._hypotests import *
 from ._rvs_sampling import rvs_ratio_uniforms, NumericalInverseHermite
-from ._page_trend_test import page_trend_test
+from ._page_trend_test_ import page_trend_test
 from ._mannwhitneyu import mannwhitneyu
 
 __all__ = [s for s in dir() if not s.startswith("_")]  # Remove dunders.
diff --git a/scipy/stats/_page_trend_test.py b/scipy/stats/_page_trend_test_.py
similarity index 100%
rename from scipy/stats/_page_trend_test.py
rename to scipy/stats/_page_trend_test_.py
-- 
2.25.1

